keras_nlp/applications/bert.py

```diff
@@ -198,3 +199,46 @@ def get_config(self):
             }
         )
         return config
+
+
+class BertClassifier(keras.Model):
+    """Classifier model with BertEncoder."""
+
+    # TODO(jbischof): figure out initialization default
```

Comment:
can we pull the initializer from the bert model by default?

Reply:
I ended up passing through default initializer args for a `Dense` layer. WDYT?

Reply:
Discussed offline to use truncated normal following the Bert paper and not to expose initialization for now.